frontend/lib/Repository/AdminRepository/groups_repository.dart

```diff
@@ -89,4 +91,61 @@ class GroupsRepository {
     }
     return Future.value(GroupSummary(-1, ''));
   }
+
+  Future<GroupEndpointsData> getEndpointsForGroup(int groupId) async {
+    final AuthResponse authResponse = await userGateway.getFromMemory();
+
+    if (authResponse.success) {
+      final String token = authResponse.tokens!.accessToken;
+      _client.options.headers["Authorization"] = "Bearer $token";
+
+      try {
+        final Response response = await _client.get(
+          backendURL + groupEndpointsUrl,
+          queryParameters: {"groupId": groupId.toString()},
+        );
+        if (response.statusCode == 200) {
+          final BackendResponse backendResponse =
+              BackendResponse.fromJson(response.data);
+          if (backendResponse.error == "") {
+            return GroupEndpointsData(
+              backendResponse.data["groupId"],
+              backendResponse.data["groupName"],
+              {
+                for (var e in backendResponse.data["endpoints"])
+                  e["label"]: EndpointForGroup.fromJson(e)
+              },
+            );
+          }
```

Comment:
Czy `GroupEndpointsData` nie powinno mieć funkcji `fromJson` i tam nie powinno znaleźć się to mappowanie?